stub.py

```diff
@@ -0,0 +1,94 @@
+"""Stub class"""
+
+from abc import abstractmethod
+from enum import Enum
+
+from pydantic.v1 import BaseModel
+
+
+class TaskType(str, Enum):
+    FDTD = "FDTD"
+    NORMAL = "NORMAL"
+    MODE_SOLVER = "MODE_SOLVER"
+    HEAT = "HEAT"
+
+
+class Stub(BaseModel):
+    @abstractmethod
+    def from_file(self, file_path: str) -> "Stub":
+        @abstractmethod
+        def from_file(self, file_path: str) -> "Stub":
```

Comment:
is this a mistake? (two nested `from_file()` functions?)

Reply:
yes, a mistake, modified.